### PR TITLE
reconciled the port for sonar.host to match sonar.yml

### PR DESCRIPTION
**Pull request by somayaj**

### CODE REVIEW

**Comment by pascalgrimaud** [comment]
Thanks for your contribution
But I think the correct fix would be to update the sonar.yml instead, to be consistent with what the generator-jhipster should generate: https://github.com/jhipster/generator-jhipster/blob/main/generators/server/templates/src/main/docker/sonar.yml.ejs

I think it's because this project is not sync with last version of JHipster

**Comment by somayaj** [comment]
> Thanks for your contribution But I think the correct fix would be to update the sonar.yml instead, to be consistent with what the generator-jhipster should generate: https://github.com/jhipster/generator-jhipster/blob/main/generators/server/templates/src/main/docker/sonar.yml.ejs
> 
> I think it's because this project is not sync with last version of JHipster

Yes, fixed. please check the latest commit.

File: src/main/docker/sonar.yml [hunk 1]
```diff
@@ -3,5 +3,5 @@ services:
   jhonline-sonar:
     image: sonarqube:8.2-community
     ports:
-      - 9000:9000
+      - 9001:9001
```
Comment (pascalgrimaud) [inline src/main/docker/sonar.yml:6]: It should be 9001:9000

**Review by pascalgrimaud — CHANGES_REQUESTED** [review]

File: src/main/java/io/github/jhipster/online/domain/deserializer/YoRCDeserializer.java [hunk 2]
```diff
@@ -106,7 +113,9 @@ public YoRC deserialize(JsonParser jp, DeserializationContext ctxt) throws IOExc
             .hasProtractor(hasProtractor)
             .hasGatling(hasGatling)
             .hasCucumber(hasCucumber)
-            .selectedLanguages(languages);
+            .selectedLanguages(languages)
+            .creationDate(Instant.parse(creationDate));
```
Comment (pascalgrimaud) [inline src/main/java/io/github/jhipster/online/domain/deserializer/YoRCDeserializer.java:117]: Not sure about that. I think it will fail if creationDate is ""
And this change should be done in another PR :)

Comment (somayaj) [reply src/main/java/io/github/jhipster/online/domain/deserializer/YoRCDeserializer.java:117]: I know but I think was eager to fix this one too. My apologies.

Comment (somayaj) [reply src/main/java/io/github/jhipster/online/domain/deserializer/YoRCDeserializer.java:117]: can you cherry pick the sonar one and leave this as is? I will use it for a new PR then.

Comment (somayaj) [reply src/main/java/io/github/jhipster/online/domain/deserializer/YoRCDeserializer.java:117]: > Not sure about that. I think it will fail if creationDate is "" And this change should be done in another PR :)

Resolved. Please check the last commit.

**Review by pascalgrimaud — APPROVED** [review]

**Comment by somayaj** [comment]
when will this be merged?

**Comment by pascalgrimaud** [comment]
When the CI passes.
It seems to be an error with prettier. Can you have a look plz?

**Comment by pascalgrimaud** [comment]
Thanks for your contribution @somayaj :)
